R/length.R

```diff
@@ -66,6 +72,7 @@
 xportr_length <- function(.df,
                           metadata = NULL,
                           domain = NULL,
+                          length = "metadata",
                           verbose = getOption("xportr.length_verbose", "none"),
                           metacore = deprecated()) {
   if (!missing(metacore)) {
```

Comment:
Suggestion to use `match.args` just like other R functions (check `?stats::cor`)

The first element becomes the default value

```suggestion
                          length = "metadata",
                          verbose = getOption("xportr.length_verbose", "none"),
                          metacore = deprecated()) {
  length <- match.args(length)
  if (!missing(metacore)) {
```

Documentation will look like this:

![image](https://github.com/atorus-research/xportr/assets/211358/b091ad79-cc62-4876-951f-4a589fdc9048)